src/QueryFilterHelpers.ts

```diff
@@ -207,38 +217,55 @@ export function interpretExpression(
   parameters: any,
   patient: R4.IPatient
 ): AnyFilter {
+  let returnFilter: AnyFilter = {
+    type: ''
+  };
   switch (expression.type) {
     case 'Equal':
-      return interpretEqual(expression as ELMEqual, library);
+      returnFilter = interpretEqual(expression as ELMEqual, library);
+      break;
     case 'Equivalent':
-      return interpretEquivalent(expression as ELMEquivalent, library);
+      returnFilter = interpretEquivalent(expression as ELMEquivalent, library);
+      break;
     case 'And':
-      return interpretAnd(expression as ELMAnd, library, parameters, patient);
+      returnFilter = interpretAnd(expression as ELMAnd, library, parameters, patient);
+      break;
     case 'Or':
-      return interpretOr(expression as ELMOr, library, parameters, patient);
+      returnFilter = interpretOr(expression as ELMOr, library, parameters, patient);
+      break;
     case 'IncludedIn':
-      return interpretIncludedIn(expression as ELMIncludedIn, library, parameters);
+      returnFilter = interpretIncludedIn(expression as ELMIncludedIn, library, parameters);
+      break;
     case 'In':
-      return interpretIn(expression as ELMIn, library, parameters);
+      returnFilter = interpretIn(expression as ELMIn, library, parameters);
+      break;
     case 'Not':
-      return interpretNot(expression as ELMNot);
+      returnFilter = interpretNot(expression as ELMNot);
+      break;
     case 'GreaterOrEqual':
-      return interpretGreaterOrEqual(expression as ELMGreaterOrEqual, library, parameters, patient);
+      returnFilter = interpretGreaterOrEqual(expression as ELMGreaterOrEqual, library, parameters, patient);
+      break;
     default:
       console.error(`Don't know how to parse ${expression.type} expression.`);
       // Look for a property (source attribute) usage in the expression tree. This can denote an
       // attribute on a resource was checked but we don't know what it was checked for.
       const propUsage = findPropertyUsage(expression, expression.localId);
+
       if (propUsage) {
         return propUsage;
       }
+      return {
+        type: 'unknown',
+        localId: expression.localId
+      };
```

Comment:
Actually tacking on to the above comment. This could be done above when the `returnFilter` is filled out before the switch statement.